backend/src/discovita/service/coach/prompts/shared/initial_message.md

```diff
@@ -0,0 +1,72 @@
+💡 Instead of stressing about money, you become **a Wealth Architect who builds abundance with ease.**  
+💡 Instead of trying to "be more confident," you become **a Confident Icon who radiates self-assurance effortlessly.**
+
+When you **define who you are,** action becomes **automatic.**
+
+## Why Identity First? Because It Shapes Everything Downstream.
+
+We are going to **position you in the world intentionally** by designing an **array of identities** that define:
+✅ **How you show up**  
+✅ **How you move through challenges**  
+✅ **How you build your relationships**  
+✅ **How you experience life itself**
+
+This process isn't about **external labels or expectations.**  
+✅ **Not your dad's values.**  
+✅ **Not your partner's expectations.**  
+✅ **Not society's definition of success.**  
+
+This is about **what lights you up inside.**
+
+When we find the **right identities for you,** you will **feel it.**
+
+💡 It will give you a **surge of energy**—like a puzzle piece snapping into place in your heart.  
+💡 It will **clarify your choices**—because when you know who you are, the right actions become obvious.  
+💡 It will shift you from **forcing action** to **living in alignment.**
+
+## Bringing the External World Inside: Who Are You in Relationship to Your Life?
+
+Instead of seeing different areas of life—**work, fitness, relationships, wealth, health**—as external buckets that you have to manage, we are flipping the perspective:
+
+💡 **Who are you in relationship to your health?** → A Warrior, a Vitality Seeker, an Athlete.  
+💡 **Who are you in relationship to your wealth?** → A Wealth Architect, an Abundant Creator, a Master Investor.  
+💡 **Who are you in relationships?** → A Devoted Partner, a Magnetic Presence, a Deep Connector.
+
+By making this shift, we move from **obligation to ownership.**
+
+Your external world no longer dictates your actions—**your identity does.**
+
+## Let's Design Your Identity Blueprint
+
+Right now, we're exploring—so don't get **stuck in semantics.** You might start with a concept, and later, we'll refine it into a **powerful noun that you fully own.**
+
+💡 If something feels **off, we tweak it.**  
+💡 If something feels **flat, we expand it.**  
+💡 If something **sparks something inside you**, we lean into it.
+
+This process is about finding **the words that make you feel free, powerful, and limitless.**
+
+## Final Thought: The Power of Identity in Daily Life
+
+When this process is complete, you will never have to **force** action again. You will wake up and know:
+
+🔥 *"I am a Warrior, so I train."*  
+🔥 *"I am a Wealth Architect, so I manage my money wisely."*  
+🔥 *"I am a Confident Icon, so I carry myself with certainty."*
+
+And on the days when you slip? That's okay. You will have this **identity map stored in your technology tool, ready to guide you back on track, any time, any day.**
+
+Today, we take the first step.
+
+🚀 **Are you ready?**
```

Comment:
If possible, let's get this removed from front-end so there's only one copy.  Can be follow-up TD & PR.